projects/storefrontlib/src/recipes/storefront-foundation.module.ts

```diff
@@ -1,24 +1,40 @@
 import { NgModule } from '@angular/core';
 import {
   AuthModule,
+  CartModule,
+  CheckoutModule,
+  CmsModule,
   ConfigModule,
+  GlobalMessageModule,
   I18nModule,
+  ProcessModule,
+  ProductModule,
   provideConfigFromMetaTags,
   StateModule,
+  UserModule,
 } from '@spartacus/core';
 import { RoutingModule } from '../cms-structure/routing/routing.module';
 import { LayoutModule } from '../layout/layout.module';
+import { StorefrontComponent } from '../layout/main/storefront.component';
 
 @NgModule({
   imports: [
-    StateModule,
+    StateModule.forRoot(),
     AuthModule.forRoot(),
     ConfigModule.forRoot(),
-    RoutingModule,
+    RoutingModule.forRoot(),
     I18nModule.forRoot(),
+    CmsModule.forRoot(),
+    GlobalMessageModule.forRoot(),
+    ProcessModule.forRoot(),
+    CartModule.forRoot(),
+    CheckoutModule.forRoot(),
+    UserModule.forRoot(),
+    ProductModule.forRoot(),
 
     LayoutModule,
   ],
   providers: [...provideConfigFromMetaTags()],
+  exports: [StorefrontComponent],
```

Comment:
The foundation module should actually not export the storefront module, since the intend is that this module only contains the bare minimum.

Reply:
Fixed